commands/time.js

```diff
@@ -0,0 +1,42 @@
+/* Usage: .time [zone]
+ * Ex: .time Asia/Beijing, .time JST
+ */
+
+import {Command} from './command.js';
+import timezone from 'timezone/loaded';
+
+export class Time extends Command {
+  message(from, to, text, message) {
+    let timezoneMatch = text.match(/^[.!]time (.*)/);
+    return new Promise((resolve, reject) => {
```

Comment:
Oh, wrapping the if is interesting... didn't even think of that. I'm gonna adopt that model, it'll clean my code up a bit!